WalletWasabi/WabiSabi/Models/DecompositionAlgs/Coin.cs

```diff
@@ -0,0 +1,40 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public record Coin : IComparable
+	{
+		public Money Amount { get; init; } = Money.Zero;
+		public const int OutputVbytes = 31;
+		public const decimal InputVbytes = 68.5m;
```

Comment:
The class could be improved by adding a `ScriptPubKeyType` property.

Reply:
actually `Coin` (which should be renamed) is simpler than the `NBitcoin.Coin` class, it's just an int that needs to *not* have int equality, since you sets need to contain multiple ones with the same value sometimes, a reference type wrapper around an int.

Reply:
ACK, removed the Coin class.